doc/source/install.rst

```diff
@@ -22,14 +22,14 @@ They can be installed individually:
 
 .. code-block:: bash
 
-    $ pip install satpy[viirs_sdr]
+    $ pip install "satpy[viirs_sdr]"
 
 Or all at once, although this isn't recommended due to the large number of
 dependencies:
 
 .. code-block:: bash
 
-    $ pip install satpy[all]
+    $ pip install "satpy[complete]"
```

Comment:
```suggestion
    $ pip install "satpy[all]"
```
We don't have 'complete' defined in our `setup.py` file.

Reply:
To give more context to what @mraspaud is saying @raybellwaves: The thing inside the brackets is actually referencing items mentioned in the "extra_requires" parameter of the setup.py file. In our setup.py we call this special "all extra dependencies"-type thing "all". In other projects it sounds like they call it "complete".

Reply:
But complete works which is why I'm confused and it gets further than all. Anyway I'll revert back to all and move this to a new issue

Reply:
See my comment on your new issue #1350. `complete` and `all` aren't features of pip, they are manually coded "extras" that dask and satpy have included for the users convenience. They call theirs complete, we call ours all.